test/suite/__snapshots__/extension.test.js.snap

```diff
@@ -3,12 +3,12 @@
 exports[`Extension Test Suite Format document using .prettierrc 1`] = `
 "/* eslint-disable */
 const options = {
-    first: "hello, world",
+    first: 'hello, world',
```

Comment:
This indicates that there is a bug as the snapshots shouldn't change between versions